RuminantFarmSystems/RuFaS
# Gnu save var names formats

This PR seeks to address issue #813 in which SMEs are not able to use the variable_names.txt in the capacity they desire to.

The Output Manager had an existing array of formatting options which were previously not easily accessed. At Pooya's suggestion, these formatting options are being tested here as possible solutions to address #813. To help with this process, this PR:

1. Adds gnu args for the 3 formatting options: "verbose", "inline", and "block." The previous default setting for formatting was "verbose" and was kept as the default here as well.
~~2. Adds a function to determine which of the 3 options the user selected and passes that value to the main.py run_rufas() function which eventually gets it to OutputManager for formatting the variable_names.txt file.~~
3. Updates unit testing.

To test out these new format_option gnu args:
1. Run a simulation specifying a formatting option. E.g. for the "inline" formatting option, use the command-line `python main.py -f=inline`. For "block" use `python main.py -f=block`. The default is the "verbose" option so to see this version, the user can either use `python main.py -f=verbose` or `python main.py`.
2. After each simulation, look at the resulting variable_names.txt file in the output folder and see if any of them will fulfill their needs.

If none of these formats fulfills the needs of the SMEs, please specify what you would like and we can work on a different format that does meet those needs.
